src/actions/classifier.js

```diff
@@ -1,14 +1,30 @@
 import apiClient from 'panoptes-client/lib/api-client'
-import { isNil } from 'ramda'
+import { forEach, isEmpty, isNil } from 'ramda'
 import { setState } from '../actions/index'
+import { getAuthUser } from '../actions/auth'
 import { Actions } from 'react-native-router-flux'
 import { Alert } from 'react-native'
 
 export function startNewClassification(workflowID) {
-  return (dispatch) => {
+  return (dispatch, getState) => {
+    dispatch(setState('classifier.isFetching', true))
     dispatch(setState('loadingText', 'Loading Workflow...'))
     dispatch(setState('classifier.currentWorkflowID', workflowID))
     dispatch(fetchWorkflow(workflowID)).then(() => {
+      if (getState().classifier.tutorial[workflowID] !== undefined) {
+        return
+      }
+      return dispatch(fetchTutorials(workflowID))
+    }).then(() => {
+      if (getState().classifier.project[workflowID] !== undefined) {
+        return
+      }
+      return dispatch(fetchProject(workflowID))
+    }).then(() => {
+      return dispatch(setupProjectPreferences(workflowID))
+    }).then(() => {
+      return dispatch(setNeedsTutorial())
+    }).then(() => {
       dispatch(setState('classifier.isFetching', false))
     }).catch(() => {
       Alert.alert('Error', 'Sorry, but there was an error loading this workflow.  Please try again later.',
```

Comment:
When I am signed in, I keep getting to this `.catch()` when trying to open Planet Four. However, there is no problem when I am signed out. It could be an issue with my environment though?